packages/contract-wrappers/src/utils/calldata_optimization_utils.ts

```diff
@@ -0,0 +1,44 @@
+import { SignedOrder } from '@0x/types';
+import * as _ from 'lodash';
+
+const NULL_BYTES = '0x';
+
+export const calldataOptimizationUtils = {
```

Comment:
This doesn't belong in `contract-wrappers`. Why did it end up in this package?

Reply:
It used to live in the there for the Forwarder contract wrapper

Reply:
Right, but since it's not used by the Forwarder contract wrapper anymore, I'd vote to move it to wherever it's actually being used.

Reply:
@abandeali1 @hysz is this still useful with the optimised ABI encoder?

Reply:
@dekz this is no longer necessary as it optimized encoding happens by default.

Reply:
removed 🔥